src/crud.py

```
@@ -218,16 +369,37 @@ async def create_session(
+        app_id: ID of the app
+        user_id: ID of the user
+        
+    Returns:
+        The created session
+        
+    Raises:
+        ResourceNotFoundException: If the user does not exist
+    """
+    try:
+        # This will raise ResourceNotFoundException if user not found
+        honcho_user = await get_user(db, app_id=app_id, user_id=user_id)
```

Ruff (0.8.2)

Local variable `honcho_user` is assigned to but never used

Remove assignment to unused variable `honcho_user`

(F841)